docs/pages/hooks/useCreateNFT.mdx

```diff
@@ -104,14 +106,15 @@ Arguments:
 
 ```tsx
 {
-  standard: Standard,                           // can be Standard.ERC721 or Standard.ERC1155
+  chainId: Number,                              // id of the chain to mint on
+  collectionAddress: string,                    // address of the collection to mint on
   name: string,                                 // name of the NFT
   description: string,                          // description of the NFT
   content: File,                                // content file uploaded by the user
   preview?: File,                               // preview in the case of private or animation content uploaded by user
   isAnimation: boolean,                         // set to true if content file is a video. Require to set preview
   isPrivate: boolean,                           // set to true if content is private. Require to set preview.
```

Comment:
Those two could be optional with a default to make the API easier to understand for beginner.
```suggestion
  isAnimation: boolean = false,                         // set to true if content file is a video. Require to set preview
  isPrivate: boolean = false,                           // set to true if content is private. Require to set preview.
```

Reply:
https://github.com/liteflow-labs/liteflow-js/issues/104